pyaedt/application/Design.py

```diff
@@ -354,7 +413,52 @@ def update(self):
 
 
 class Design(object):
-    """Class Design. Contains all functions and objects connected to the active Project and Design"""
+    """Design class. 
+    
+    This class contains all functions and objects connected to the active project and design.
+    
+    Parameters
+    ----------
+    design_type : str
+        Type of the design.
+    project_name : str, optional
+        Name of the project to select or the full path to the project
+        or AEDTZ archive to open. The default is ``None``, in which
+        case an attempt is made to get an active project. If no 
+        projects are present, an empty project is created.
+    design_ame : str, optional
+        Name of the design to select. The default is ``None``, in 
+        which case an attempt is made to get an active design. If no
+        designs are present, an empty design is created.
+    solution_type : str, optional
+        Solution type to apply to the design. The default is
+        ``None``, in which case the default type is applied.
+    specified_version: str, optional
+        Version of AEDT to use. The default is ``None``, in which case
+        the active version or latest installed version is used.
+    NG : bool, optional
+        Whether to run AEDT in the non-graphical mode. The default 
+        is ``False``, which launches AEDT in the graphical mode.  
+    AlwaysNew : bool, optional
+        Whether to launch an instance of AEDT in a new thread, even if
+        another instance of the ``specified_version`` is active on the
+        machine. The default is ``True``.
+    release_on_exit : bool, optional
+        Whether to release AEDT on exit. The default is ``False``.
+   
+    Examples
```

Comment:
Example in this class can be removed because this is an internal class called from application classes (like Hfss, Maxwell, etc...)